_posts/2018-09-27-SpringOne-Platform.md

```diff
@@ -0,0 +1,151 @@
+
+Watch the talk <a href="https://www.youtube.com/watch?v=1ezNJvajUU0&feature=youtu.be&t=9m34s" target="_blank">here</a>.
+
+### First-timers-only issues
+
+Because it can be daunting for people to enter a community they are creating "first-timers-only" issues on the Spring Boot project, making it easier for newcomers to get connected.
+While working on these issues developers of the Spring Boot project will come alongside you to teach you to contribute back, help you and coach you.
+This way you will get to become part of the Spring community.
+
+Watch the talk <a href="https://www.youtube.com/watch?v=1ezNJvajUU0&feature=youtu.be&t=10m39s" target="_blank">here</a>.
+
+### Pivotal Tracker: Maestro
+
+> What happens when your organization scales, and you end up with lots of fast-moving developers?
+> It's really, really easy to get misaligned.
+> And when you get misaligned, it's really easy to ship the wrong thing.
+
+That's why Pivotal has built a tool to help teams articulate and align around business outcomes to give organizations the high-level view so they can assure that everyone is aligned and delivering value.
+
+Watch the talk <a href="https://www.youtube.com/watch?v=1ezNJvajUU0&feature=youtu.be&t=11m43s" target="_blank">here</a>.
+
+### Pivotal Act
+
+> "Pivotal Act is a program that partners with humanitarian organizations and charities to identify, design, and develop practical solutions to pressing challenges around the world."
+
+Together with developers, designers, and engineers from Pivotal Labs, Pivotal is going to apply the same methodology they use in other engagements with clients but tailored to the needs of the humanitarian and social impact sectors instead of just donating technology or funds.
+This way charitable and nonprofit organisations can use the present-day technologies and put them directly to use through the partnership with Pivotal, while building up the organisation's technology capabilities so they can continue their work after the engagement ends.
+
+If you want to find out more, just visit <a href="https://pivotal.io/act" target="_blank">https://pivotal.io/act</a>.
+
+You can watch the announcement <a href="https://pivotal.io/act" target="_blank">here</a>.
+
+### R2DBC
+
+When using the reactive model, people were limited to NoSQL databases because those were the only ones that had a driver which supports things like streams and backpressure.
+During the keynote Oliver Gierke announced the R2DBC project.
+This project currently consists out of a client, an SPI and a PostgreSQL implementation to bring the reactive capabilities to SQL databases.
+It also has an adapter to support the OpenJDK incubator project "ADBA", a non-blocking database access API that Oracle is proposing as a Java standard.
```

Comment:
Maybe also state that it is not yet production ready, but promising
see https://twitter.com/nebhale/status/1044962921577697280